packages/ui-components/src/components/Chart/Legend.js

```diff
@@ -118,8 +120,12 @@ export const getCartesianLegend = ({ chartConfig, onClick, getIsActiveKey, isExp
           isExporting={isExporting}
           style={{ textDecoration: getIsActiveKey(value) ? '' : 'line-through' }}
         >
-          <Box style={{ background: color }} />
-          <Text>{getDisplayValue(chartConfig, value)}</Text>
+          <Tooltip title="Click to filter data" placement="top" arrow>
```

Comment:
Nicely snuck in!

Reply:
Should we have the same on the pie legend?

Reply:
No I don't think so. Clicking the pie legend doesn't filter the data (neither on existing and new version of charts). I guess that was never added because it wouldn't really work on pie charts.